app/src/controllers/SmartContractWalletController.js

```diff
@@ -69,6 +131,92 @@ export default class SmartContractWalletController {
     @param txId {number} - the tx's Id
     @returns - rawTx {string}
   */
+
+  async getContractNonce() {
+    let nonceURL = config.biconomy.concat('/api/v1/wallet/getContractNonce')
+    let signer = this.getSelectedEOA()
+
+    let response
+    try {
+      response = await get(`${nonceURL}?signer=${signer}`, {
+        headers: {
+          'x-wallet-key': config.biconomyKey[KOVAN]
+        }
+      })
+      console.log(response)
+      return response.nonce
+    } catch (e) {
+      log.error(e)
+      return []
+    }
+  }
+
+  async sendTransaction(biconomyReqObj) {
+    let relayUrl = config.biconomy.concat('/api/v1/wallet/relay')
+
+    let response
+    try {
+      response = await post(relayUrl, biconomyReqObj, {
+        headers: {
+          'x-wallet-key': config.biconomyKey[KOVAN],
+          'Content-Type': 'application/json; charset=utf-8'
+        }
+      })
+      console.log(response)
+      return response
+    } catch (e) {
+      log.error(e)
+      return []
+    }
+  }
+
+  async messageToSendTx(to, value, userContractWallet, nonce) {
+    let systemInfo = await get(`${config.biconomy}/api/v2/meta-tx/systemInfo?networkId=${KOVAN_CODE}`)
+
+    let metaInfo = {
+      contractWallet: userContractWallet
+    }
+
+    let relayerPayment = {
+      token: config.DEFAULT_RELAYER_PAYMENT_TOKEN_ADDRESS,
+      amount: config.DEFAULT_RELAYER_PAYMENT_AMOUNT
+    }
+
+    let message = {
+      from: this.getSelectedEOA().toLowerCase(),
+      to: to.toLowerCase(),
+      data: '0x0',
+      batchId: 0,
+      nonce: parseInt(nonce),
+      value: this.web3.utils.toHex(value),
+      txGas: '21000',
+      expiry: 0,
+      baseGas: 0,
+      metaInfo: metaInfo,
+      relayerPayment: relayerPayment
+    }
+
+    let domainData = {
+      name: config.eip712DomainName,
+      version: config.eip712SigVersion,
+      verifyingContract: config.eip712VerifyingContract,
```

Comment:
Set this value from systemInfo.relayHubAddress to get updated value as per the network selected.